commit 933ca791a7a9dbd36f58a502181c5822a3c5cadf
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sun Apr 21 16:52:53 2019 +0200

    shutdown: Fix static link
    
    * shutdown/Makefile (HURDLIBS): Add iohelp ihash.

diff --git a/shutdown/Makefile b/shutdown/Makefile
index ced79702..8c219c4c 100644
--- a/shutdown/Makefile
+++ b/shutdown/Makefile
@@ -19,7 +19,7 @@ dir := shutdown
 makemode := server
 
 SRCS = shutdown.c acpi_shutdown.c
-HURDLIBS = ports shouldbeinlibc trivfs
+HURDLIBS = ports shouldbeinlibc trivfs iohelp ihash fshelp
 target = shutdown
 MIGSTUBS = shutdownServer.o
 MIGSFLAGS = -imacros $(srcdir)/mig-mutate.h
diff --git a/shutdown/Makefile b/shutdown/Makefile
index 2d2593c0..73813dab 100644
--- a/shutdown/Makefile
+++ b/shutdown/Makefile
@@ -20,6 +20,7 @@ makemode := server
 
 SRCS = shutdown.c acpi_shutdown.c
 HURDLIBS = ports shouldbeinlibc trivfs iohelp ihash fshelp
+LDLIBS = -lpthread
 target = shutdown
 MIGSTUBS = shutdownServer.o
 MIGSFLAGS = -imacros $(srcdir)/mig-mutate.h
